src/pages/JobDescription/components/JobDescriptionItem.jsx

```diff
@@ -0,0 +1,41 @@
+import { useTheme } from "@emotion/react";
+import { ToggleButton, Grid, ListItem, ListItemText } from "@mui/material";
+import React from "react";
+import PropTypes from "prop-types";
+/**
+ * A component that displays a training block with course name, skills, progress, and description.
+ * @param {Object} props - The props object.
+ * @param {Object} props.JobDescription - The training object containing course name, skills, progress, description, and status.
+ * @param {string} props.mockedSkills - Prints the Skills in the toggles
+ * @returns {JSX.Element} the JobDescriptionPage Component.
+ */
+
+function JobDescriptionItem(props) {
+  const [selected, setSelected] = React.useState(false);
+  return (
+    <Grid item xs={3}>
+      <ToggleButton
+        value="check"
+        selected={selected}
+        onChange={() => {
+          setSelected(!selected);
+        }}
+        sx={{
+          height: "50px",
+          width: "150px",
+          background: useTheme().palette.primary.light,
+        }}
+      >
+        <ListItem sx={{ textAlign: "center" }}>
+          <ListItemText>{props.mockedSkills}</ListItemText>
+        </ListItem>
+      </ToggleButton>
+    </Grid>
+  );
+}
+
+JobDescriptionItem.propTypes = {
+  mockedSkills: PropTypes.string,
+};
```

Comment:
It is only one skill that is passed, so the naming doesn't fit here.
And it also shouldn't be called "mocked..."